server/test/app/ApiAuthenticationTest.java

```diff
@@ -67,10 +62,7 @@ public void apiRoutes_invalidKey() {
 
     Result result =
         doGetRequest(
-            fakeRequest(
-                    "GET",
-                    controllers.api.routes.ProgramApplicationsApiController.list("mock-program")
-                        .url())
+            fakeRequest("GET", API_URL)
```

Comment:
nit: API_URL -> AUTH_CHECK_URL

It doesn't read clearly in uses like there.

Reply:
done